Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/Iterators.dfy.expect

```diff
@@ -40,5 +31,14 @@ Iterators.dfy(461,21): Error: this invariant could not be proved to be maintaine
  Related message: loop invariant violation
 Iterators.dfy(470,4): Error: possible violation of yield-ensures condition
 Iterators.dfy(451,21): Related location: this proposition could not be proved
+Iterators.dfy(251,9): Error: decreases clause might not decrease
```

Comment:
Why did the ordering change? Seems like a regression since we've been better about keeping these in order lately.

Reply:
It's unavoidable, since we want to verify one module at a time, and when nested modules occur, the contents of different modules can be intertwined.

```dafny
method A

module Nested {
  method B
}

method C
```

We can do A,C,B or B,A,B, but we can no longer do A,B,C if we verify entire modules at a time.